provision/acc_provision/test_main.py

```diff
@@ -11,9 +11,19 @@
 import tarfile
 import json
 
+# import yaml
+import base64
+import copy
+
 from . import acc_provision
 from . import fake_apic
 
+from ruamel.yaml import YAML
+yml = YAML()
```

Comment:
If it only used in the compare function, we can define it in the function itself , rather than being global

Reply:
same for line 22-25

Reply:
It is used in two functions - compare_kube_yaml and compare_tar_plaintext thats why kept global